src/library/Uno.Material/Controls/Chips/ChipGroup.cs

```diff
@@ -95,11 +31,14 @@ public ChipGroup()
 		protected override void OnApplyTemplate()
 		{
 			base.OnApplyTemplate();
+			ApplyThumbnailTemplate();
 		}
 
 		private void OnLoaded(object sender, RoutedEventArgs e)
 		{
 			EnforceSelectionMode();
+			ApplyThumbnailTemplate();
```

Comment:
properties forwarding should be done inside PrepareContainerForOverrides
and cleaned up in ClearContainerForOverrides.

Reply:
done! Thanks for the suggestion